From 24a12092992a334556405e80cbe0505b71d2b994 Mon Sep 17 00:00:00 2001
From: David Boddie <david.boddie@qt.io>
Date: Tue, 10 Jun 2025 13:56:05 +0200
Subject: [PATCH 229/553] Doc: Fix self-linking "see also" link

Pick-to: 6.9
Change-Id: Ieb132dbd70fd0469040e6b55763df1f1b5fc4ab7
Task-number: QTBUG-137048
Reviewed-by: Paul Wicking <paul.wicking@qt.io>
(cherry picked from commit a0d6e496e4a75e27a2ea85a3694a7b1430272999)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/global/qoverload.qdoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/corelib/global/qoverload.qdoc b/src/corelib/global/qoverload.qdoc
index aaca504cfee..0108569deb5 100644
--- a/src/corelib/global/qoverload.qdoc
+++ b/src/corelib/global/qoverload.qdoc
@@ -45,6 +45,6 @@
 
     \snippet code/src_corelib_global_qglobal.cpp 54
 
-    \sa qOverload, qNonConstOverload, {Differences between String-Based
+    \sa qOverload, qConstOverload, {Differences between String-Based
     and Functor-Based Connections}
 */
-- 
2.51.2

